javascript/ql/src/Security/CWE-020/IncompleteUrlRegExp.ql

```diff
@@ -0,0 +1,68 @@
+/**
+ * @name Incomplete URL regular expression
+ * @description Using a regular expression that contains an 'any character' may match more URLs than expected.
```

Comment:
"an 'any character'" reads awkwardly. Perhaps something along the lines of "Matching a URL against a regular expression that contains an unescaped dot as part of the hostname might match more URLs than expected."